modules/virt-about-node-placement-vms.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-vms.adoc
+
+[id="virt-about-node-placement-vms_{context}"]
+= About node placement for virtual machines
+
+To ensure that virtual machines (VMs) run on appropriate nodes, you can configure node placement rules. You might want to do this if:
+
+* You have several VMs. To ensure fault tolerance, you want them to run on different nodes.
+* You have two chatty VMs. To avoid redundant inter-node routing, you want the VMs to run on the same node.
+* Your VMs require specific hardware features that are not present on all available nodes.
+* You have a pod that adds capabilities to a node, and you want to place a VM on that node so that it can use those capabilities.
+
+You can use the following rule types in the `spec` field of a `VirtualMachine` manifest:
+
+`nodeSelector`:: Allows virtual machines to be scheduled on nodes that are labeled with the key-value pair or pairs that you specify in this field. The node must have labels that exactly match all listed pairs.
+`affinity`:: Enables you to use more expressive syntax to set rules that match nodes with virtual machines. Pod affinity, pod anti-affinity, and node affinity are supported for virtual machine placement. In comparison to `nodeSelector`, affinity allows for more nuance in how the rules are applied. For example, you can specify that a rule is a preference, rather than a hard requirement, so that virtual machines are still scheduled if the rule is not satisfied.
++
```

Comment:
I don't know where this should go in the document, but it might be nice to emphasize that affinity placement rules apply only during scheduling (like their names say "during scheduling". It's literal). OpenShift does not currently have the ability to reschedule already-running workloads if a constraint would later be violated.